### PR TITLE
BXMSDOC-8443 PIM docs incorrect for 7.12

### PR DESCRIPTION
**Pull request by emmurphy1**
HTML draft:
http://file.ork.redhat.com/~emmurphy/BXMSDOC-8443-pam/#process-instance-migration-installing-service-proc

### CODE REVIEW

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc [hunk 1]
```diff
@@ -9,100 +9,92 @@ You can use the process instance migration (PIM) service to create, export and e
...
+. Create the database schema:
+
+.. Download the schema creation files:
 +
-[source,subs="attributes+"]
+[source]
 ----
-$ java -jar {PRODUCT_FILE}-process-migration-service-standalone.jar -s./config.yml
+$ curl https://raw.githubusercontent.com/kiegroup/process-migration-service/main/ddl-scripts/postgresql/postgresql-quartz-schema.sql -o postgresql-quartz-schema.sql
+$ curl https://raw.githubusercontent.com/kiegroup/process-migration-service/main/ddl-scripts/postgresql/postgresql-pim-schema.sql -o postgresql-pim-schema.sql
```
Comment (gmunozfe) [inline doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc]: why download them from github? (in fact, main branch could be misaligned from the PRODUCT-FILE version)

Those scripts are present already in the zip under `process-migration/ddl-scripts/postgres` path.

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc [hunk 2]
```diff
@@ -9,100 +9,92 @@ You can use the process instance migration (PIM) service to create, export and e
...
-        password: secret
-      identity-mapping:
-        search-base-dn: ou=users,dc=example,dc=com
-        rdn-identifier: uid
-        attribute-mappings:
-          from: uid
-          filter: uniqueMember
-          filter-base-dn: ou=groups,dc=example,dc=com
+$ psql -U jboss -h localhost -d rhpam7 -f ~/Downloads/postgresql-quartz-schema.sql
+$ psql -U jboss -h localhost -d rhpam7 -f ~/Downloads/postgresql-pim-schema.sql
```
Comment (gmunozfe) [inline doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc]: Same here, point to `process-migration/ddl-scripts/postgres/postgresql-quartz-schema.sql` and `process-migration/ddl-scripts/postgres/postgresql-pim-schema.sql`.

Other minor comment is to put user and host between brackets to indicate that the customer should use the corresponding ones in its deployment, wdyt?

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc [hunk 3]
```diff
@@ -9,100 +9,92 @@ You can use the process instance migration (PIM) service to create, export and e
...
-      attribute-mappings:
-        0:
-          index: 2
-          to: groups
+  datasource:
+    db-kind: postgresql
+    jdbc:
+      url: jdbc:postgresql://localhost:5432/rhpam7
+    username: jboss
+    password: jboss
```
Comment (gmunozfe) [inline doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc]: Same here, or add a comment that's just an example that may be tuned to the corresponding configuration for the user

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc [hunk 4]
```diff
@@ -9,100 +9,92 @@ You can use the process instance migration (PIM) service to create, export and e
...
+[source]
 ----
-kieservers:
-  - host: http://kieserver1.example.com:8080/kie-server/services/rest/server
-    username: admin
-    password: admin123
-  - host: https://kieserver2.example.com:8443/kie-server/services/rest/server
-    username: foo
-    password: bar
+$ java -jar -Dquarkus.launch.rebuild=true -Dquarkus.datasource.db-kind=postgresql quarkus-app/quarkus-run.jar
```
Comment (gmunozfe) [inline doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc:53]: This command has a similar output to this:
INFO  [io.qua.dep.QuarkusAugmentor] (main) Quarkus augmentation completed in 2657ms

**Review by gmunozfe — CHANGES_REQUESTED** [review]
DDL Scripts are already present in the delivered zip, not need to download them from Github (may be different).
Other minor comments.

File: doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc (hunk 3 above)
Comment (emmurphy1) [reply doc-content/enterprise-only/processes/process-instance-migration-installing-service-proc.adoc]: Updated.

**Pull request merged**